x/lockup/types/msgs_test.go

```diff
@@ -95,10 +95,224 @@ func TestMsgLockTokens(t *testing.T) {
 	}
 }
 
-// TODO: Complete table driven tests for the remaining messages
+func TestMsgBeginUnlockingAll(t *testing.T) {
+	appParams.SetAddressPrefixes()
+	pk1 := ed25519.GenPrivKey().PubKey()
+	addr1 := sdk.AccAddress(pk1.Address()).String()
+	invalidAddr := sdk.AccAddress("invalid")
+
+	createMsg := func(after func(msg MsgBeginUnlockingAll) MsgBeginUnlockingAll) MsgBeginUnlockingAll {
+		properMsg := MsgBeginUnlockingAll{
+			Owner: addr1,
+		}
+
+		return after(properMsg)
+	}
+
+	msg := createMsg(func(msg MsgBeginUnlockingAll) MsgBeginUnlockingAll {
+		// Do nothing
+		return msg
+	})
+
+	require.Equal(t, msg.Route(), RouterKey)
+	require.Equal(t, msg.Type(), "begin_unlocking_all")
+	signers := msg.GetSigners()
+	require.Equal(t, len(signers), 1)
+	require.Equal(t, signers[0].String(), addr1)
+
+	tests := []struct {
+		name       string
+		msg        MsgBeginUnlockingAll
+		expectPass bool
+	}{
+		{
+			name: "proper msg",
+			msg: createMsg(func(msg MsgBeginUnlockingAll) MsgBeginUnlockingAll {
+				// Do nothing
+				return msg
+			}),
+			expectPass: true,
+		},
+		{
+			name: "invalid owner",
+			msg: createMsg(func(msg MsgBeginUnlockingAll) MsgBeginUnlockingAll {
+				msg.Owner = invalidAddr.String()
+				return msg
+			}),
+			expectPass: false,
```

Comment:
```suggestion
```

expectPass defaults to false, so every occurrence of false can be removed from all the tests to make the tests shorter.